File: Java/src/com/arvindhvelrajan/UI_Functions.java
package com.arvindhvelrajan;

/**
 * @author arvindhvelrajan
 * @code This class is meant to keep all functions necessary for smooth UI in the command line when running the created classes.
 */
public class UI_Functions
{
	private static int callCount = 0;

	public static void resetCallCount()
	{
		callCount = 0;
	}
	public static void printFunction(String functionName)
	{
		if(callCount != 0)
		{
			System.out.println();
			System.out.println();
		}
		System.out.println("Function Name: " + functionName + "()");
		System.out.println();
		callCount++;
	}
}

File: Python/UI_Functions.py

class UI_Functions:
    call_count = 0

    @classmethod
    def reset_callCount(cls):
        cls.call_count = 0

    @classmethod
    def print_function(cls, functionName):
        if cls.call_count == 0:
            print("\n\n")
        print("Function Name: " + functionName + "()\n")
        cls.call_count += 1
